vulture/core.py

```diff
@@ -184,6 +184,18 @@ def scavenge(self, paths):
             else:
                 self.scan(module_string, filename=module)
 
+            whitelist_names = ['stdlib.py']
+            for name in whitelist_names:
+                path = os.path.join('whitelists', name)
+                if exclude(path):
+                    self.log('Excluded Whitelist:', path)
+                else:
+                    module_data = pkgutil.get_data('vulture', path)
+                    if module_data is None:
+                        sys.exit('Error: Please use "python -m vulture.core".')
```

Comment:
Previously it was possible to run "python core.py". Now this is not possible anymore.